tests/stability/test_stability.py

```diff
@@ -0,0 +1,28 @@
+import sys
+import uuid
+
+import coiled
+import dask
+from dask.distributed import Client
+
+SOFTWARE = f"dask-engineering/coiled_dist-py{sys.version_info[0]}{sys.version_info[1]}"
+
+
+def test_shuffle_stability():
+    with coiled.Cluster(
+        software=SOFTWARE,
+        name="coiled-stability-tests_" + str(uuid.uuid4()),
+        account="dask-engineering",
+        n_workers=10,
```

Comment:
Out of curiosity do we need 10 workers here? I know we use 10 for the quickstart because is a decent chunk of data, but can we run this with just `n_workers=4`?